### ISSUE TITLE
Generators seem to be available for everyone?

### ISSUE THREAD

**Issue by Seanstoppable** [mbryzek/apidoc#425]
Commons 6/7 Clients have no meaning to me, and it is slightly confusing to have them show up. Are all generators intended to be public to all organizations?

**Comment by mbryzek** [comment]
agreed - this is by design - w/ idea being we'd revisit at appropriate time

**Comment by mchimirev** [comment]
Another use of private generators would be to hide work-in-progress generators.  Android generator, for example, does not yet support unions.  Good enough for us to use with many services but not good enough to give to open source community.

**Comment by mchimirev** [comment]
Yet another issue with public generators is whoever is hosting the generator has access to api definitions for any one that uses that generator.  We host both android generators under gilt AWS account (via AWS Gateway and AWS Lambda), and therefore any time android client is generated - we could write some one else's service definition to our logs.  

In other words: give_me_a_copy_of_your_api_generator

**Comment by mkows** [comment]
I think that this issue is fixed now and can be closed (example my dummy org https://app.apibuilder.io/michal/playground-api/latest seems to expose only common generators)

**Comment by gheine** [comment]
Don't think it's resolved. A few invalid and deprecated generators where removed recently, but we're still exposing all generators to all users, rather than being able to set/configure visibility to eg. user or org only. 

IMO generators should by default be at most org visible and only after a community vote of some sort be made public, so that the list of available generators is not (accidentally or deliberately) polluted.

Also like the idea of hiding feature incomplete generators. How about we just omit them in the UI, but keep them usable via the api/cli?
